job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -91,6 +91,12 @@ paths:
         format: int32
         required: false
         description: The starting position from which to return results (useful for paging)
+      - name: sort
+        in: query
+        type: string
+        required: false
+        description: 'How to order the returned results, in the format <field>:<direction>.  Allowed values for field: JOB_ID, CREATE_DATE.  Allowed values for direction: ASCENDING, DESCENDING.'
+        default: CREATE_DATE:DESCENDING
```

Comment:
Don't you think that `createDate:desc` would be better than `CREATE_DATE:DESCENDING`?  I realize you're following an enum convention here but the fields are reported back in the payload in camelCase so I think it makes more sense for sort requests to be made in the same way (and also `:asc` and `:desc` are common abbreviations in this context).